aptos-node/src/lib.rs

```diff
@@ -689,24 +689,32 @@ pub fn setup_environment_and_start_node(
         _ => None,
     };
 
-    let jwk_consensus_runtime = if let Some(obj) = jwk_consensus_network_interfaces {
-        let ApplicationNetworkInterfaces {
-            network_client,
-            network_service_events,
-        } = obj;
-        let (reconfig_events, onchain_jwk_updated_events) = jwk_consensus_subscriptions.expect(
-            "JWK consensus needs to listen to NewEpochEvents and OnChainJWKMapUpdated events.",
-        );
-        let jwk_consensus_runtime = start_jwk_consensus_runtime(
-            network_client,
-            network_service_events,
-            vtxn_pool.clone(),
-            reconfig_events,
-            onchain_jwk_updated_events,
-        );
-        Some(jwk_consensus_runtime)
-    } else {
-        None
+    let maybe_jwk_consensus_key =
```

Comment:
Do you need to load twice? Seems same as dkg

Reply:
i don't want to, but the key is not clonable...